src/Files.App/Utils/Storage/History/StorageHistoryOperations.cs

```diff
@@ -9,7 +9,7 @@ namespace Files.App.Utils.Storage
 	public class StorageHistoryOperations : IStorageHistoryOperations
 	{
 		private IFilesystemHelpers helpers;
-		private IFilesystemOperations operations;
+		private ShellFilesystemOperations operations;
```

Comment:
Is this alright? I think not only shell one can be here.

Reply:
It's the only one

```
operations = new ShellFilesystemOperations(associatedInstance);
```